### ISSUE TITLE
3D implementation

### ISSUE THREAD

**Issue by simonjeger** [gLENTNER/CubicSplineLib#1]
Hi there,
In your documentation you talk about cubic spline interpolation in one, two, and **three** dimensions. Is the tri-cubic case implemented?

**Comment by glentner** [comment]
Nope, that was a work-in-progress. I haven't touched this code since graduate school when I created it for a project. Feel free to create a fork and add that if you like. 😄
